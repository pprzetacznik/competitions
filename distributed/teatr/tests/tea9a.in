100000000 3 134217728 40265318 1000000 16068130450042926003
