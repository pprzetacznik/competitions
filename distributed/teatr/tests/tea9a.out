2095735774040378
